### PR TITLE
CLOUDSTACK-9657: Fixed security group ipset issues with long vm name

### PR DESCRIPTION
**Pull request by jayapalu**
ipset set name  max length allowed is  31 char. If the vm name has longer name then the trimming of set has done in this fix. Also cleanup of rules are taken care.

### CODE REVIEW

File: scripts/vm/hypervisor/xenserver/vmops [hunk 1]
```diff
@@ -232,28 +233,50 @@ def deleteFile(session, args):
 
     return txt
 
+#using all the iptables chain names length to 24 because cleanup_rules groups the vm chain excluding -def,-eg
+#to avoid multiple iptables chains for single vm there using length 24
 def chain_name(vm_name):
     if vm_name.startswith('i-') or vm_name.startswith('r-'):
         if vm_name.endswith('untagged'):
             return '-'.join(vm_name.split('-')[:-1])
     if len(vm_name) > 28:
```
Comment (kishankavala) [inline scripts/vm/hypervisor/xenserver/vmops]: Check should be > 25

Comment (jayapalu) [inline scripts/vm/hypervisor/xenserver/vmops]: Updated it.

**Comment by kishankavala** [comment]
Code LGTM

**Comment by jayapalu** [comment]
tag:mergeready

**Comment by karuturi** [comment]
@jayapalu Can you run @cloudmonger tests on this?

**Comment by jayapalu** [comment]
@karuturi  CI is not running on the basic zone, it is running tests only on the advanced zone.

**Comment by rohit57** [comment]
Errors without patch 
screenshot 34
screenshot 32

Changes works fine. 
screenshot 35

LGTM

**Comment by jayapalu** [comment]
CI is not running on basic zone, we have manual testing screen shots and test LGTM is there. So marking merge ready.

**Pull request merged**